packages/hint/tests/lib/utils/schema-validator.ts

```diff
@@ -48,18 +79,23 @@ test('validate should generate the right message for a type error', (t) => {
     const result: SchemaValidationResult = validate(baseSchema, data);
 
     t.deepEqual(data, result.data);
-    t.is(result.prettifiedErrors[0], `'a' should be string.`);
+    t.is(result.prettifiedErrors[0], `a should be 'string'.`);
```

Comment:
Definitely quoted.

Single quotes are fine with me, though backticks may be even better since we're referring to code, but I don't have a strong preference.

Reply:
done!